code/modules/projectiles/gun.dm

```diff
@@ -227,7 +228,7 @@
 			if(G == src || G.weapon_weight >= WEAPON_MEDIUM)
 				continue
 			else if(G.can_trigger_gun(user))
-				bonus_spread += 24 * G.weapon_weight
+				bonus_spread += dual_wield_spread
```

Comment:
why isnt this dual_wield_spread * G.weapon_weight

Reply:
the only values of weapon weight are 1, 2, and 3. Only weapons with a weapon weight of 1 can dual wield, so G.weapon_weight is only ever equal to 1. So I removed it to improve code clarity